src/com/pentalog/pentastagiu/homework2/bank/Account.java

```diff
@@ -0,0 +1,26 @@
+package com.pentalog.pentastagiu.homework2.bank;
+//
+public class Account {
+
+    String fullName;
+    int accountNumber;
+    private double availableMoney;
+
+    void printAccountInfo(){
+
+        System.out.println("Your account has "+this.availableMoney);
+    }
+
+    public double getAvailableMoney() {
+
+        return availableMoney;
+    }
+
+    public void setAvailableMoney(double availableMoney) {
```

Comment:
pentru scenariul tau de test este ok cu implementarea asta, dar atunci cand scrii o clasa trebuie sa te gandesti ca va fi reflosita si in alte bucati de cod din proiect sau chiar in alte proiecte; din punctul asta de vedere, logica legata de withdraw/deposit ar trebui mutata aici si prin ea vei face toate validarile necesare si vei schimba in mod corespunzator availableMoney

daca cineva vrea sa se foloseasca de clasa ta in alt pachet, va crea un Account si va putea seta orice suma pozitiva va vrea prin setterul asta si logica ta de withdraw/deposit nu va fi folosita